broker-daemon/broker-rpc/wallet-service/get-trading-capacities.js

```diff
@@ -3,31 +3,41 @@ const { Big } = require('../../utils')
 /**
  * Grabs the total balance and total channel balance from a specified engine
  *
- * @param {Array<symbol, engine>} SparkSwap Payment Channel Network Engine
- * @return {Object} including symbol and capactiies for sending and receiving in active, pending, and inactive channels
+ * @param {Engine} SparkSwap Payment Channel Network Engine
+ * @param {String} symbol
+ * @param {String} amount of outstanding send capacity for the given currency
+ * @param {String} amount of outstanding receive capacity for the given currency
+ * @return {Object} including symbol and available, pending, outstanding, and inactive capacties for sending and receiving
  */
-async function getEngineTradingCapacities ([symbol, engine]) {
-  const [openChannelCapacities, pendingChannelCapacities] = await Promise.all([
+async function getCapacities (engine, symbol, outstandingSendCapacity, outstandingReceiveCapacity) {
+  const divideBy = currencies.find(({ symbol: configSymbol }) => configSymbol === symbol).quantumsPerCommon
```

Comment:
In other files, I think we've changed this to `const { quantumsPerCommon: divideBy } = currencies.find(({ symbol: configSymbol }) => configSymbol === symbol) || {}`. We will need to check `divideBy` to make sure we are not dividing by `0` below.

Reply:
forgot to add a [nit] on the top of that